.github/workflows/release_prep.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+# Set by GH actions, see
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+readonly TAG=${GITHUB_REF_NAME}
+# The prefix is chosen to match what GitHub generates for source archives.
+# This guarantees that users can easily switch from a released artifact to a source archive
+# with minimal differences in their code (e.g. strip_prefix remains the same)
+readonly PREFIX="rules_proto-${TAG}"
+readonly ARCHIVE="${PREFIX}.tar.gz"
+
+# Configuration for 'git archive'
+# see https://git-scm.com/docs/git-archive/2.40.0#ATTRIBUTES
+cat >.git/info/attributes <<EOF
```

Comment:
Is it possible to commit this as file, i.e. `.gitattributes`?

Benefit of that would be, it's more easy to locate and update.